pkgs/data/documentation/mustache/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "mustache-${version}";
+  version = "1.0.2";
+
+  outputs = [ "out" ];
```

Comment:
Nix supports multiple outputs, so if you add man output it will automatically move man pages to man output, which you can build with nix-build -A mustache.man or install with nix-env -iA nixpkgs.mustache.man, this results in smaller packages.